### ISSUE TITLE
Black screen?

### ISSUE THREAD

**Issue by Caalypsso** [mkinfrared/baldurs-gate3-mod-manager#72]
Application boots up as normal, but the left side stays stuck on "loading" and then the screen turns black after a few seconds.

**Comment by ruizruby55** [comment]
I am having the same problem

**Comment by cacoots** [comment]
Is this what y'all are seeing as well?

<img width="899" alt="Screenshot 2023-11-04 at 7 29 28 PM" src="https://github.com/mkinfrared/baldurs-gate3-mod-manager/assets/105724955/308225ec-1baf-4464-97fb-f88f728767e9">

**Comment by Caalypsso** [comment]
> Is this what y'all are seeing as well?
> 
> <img alt="Screenshot 2023-11-04 at 7 29 28 PM" width="899" src="https://user-images.githubusercontent.com/105724955/280497959-308225ec-1baf-4464-97fb-f88f728767e9.png">

Yes, it loads up like that and then the screen just turns blank.

**Comment by ghostofohio** [comment]
Did you ever figure out what the issue was?

**Comment by ruizruby55** [comment]
Yes. Run the game to the main menu then close it and reopen the manager On Nov 4, 2023, at 9:51 PM, ghostofohio ***@***.***> wrote:﻿
Did you ever figure out what the issue was?

**Comment by mkinfrared** [comment]
Duplicate of #104
